widgets/nuxeo-directory-widget-behavior.html

```diff
@@ -0,0 +1,147 @@
+<!--
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../../polymer/polymer.html">
+<link rel="import" href="../nuxeo-i18n-behavior.html">
+<link rel="import" href="../../iron-form-element-behavior/iron-form-element-behavior.html">
+<link rel="import" href="../../iron-validatable-behavior/iron-validatable-behavior.html">
+
+<script>
+  var Nuxeo = Nuxeo || {};
+  (function () {
+
+    /**
+     * `Nuxeo.DirectoryWidgetBehavior` provides helpers to build a widget to select Nuxeo vocabulary entrie(s).
+     *
+     * @polymerBehavior Nuxeo.DirectoryWidgetBehavior
+     */
+    Nuxeo.DirectoryWidgetBehavior = [Nuxeo.I18nBehavior, Polymer.IronFormElementBehavior, Polymer.IronValidatableBehavior, {
```

Comment:
Are we refactoring `nuxeo-directory-suggestion` too ?

Reply:
I'll see that in a second step certainly addressing https://jira.nuxeo.com/browse/NXP-24471